docs/docusaurus.config.js

```diff
@@ -0,0 +1,85 @@
+const lightCodeTheme = require('prism-react-renderer/themes/github');
+const darkCodeTheme = require('prism-react-renderer/themes/dracula');
+
+/** @type {import('@docusaurus/types').DocusaurusConfig} */
+module.exports = {
+  title: 'Arcus Background Jobs',
+  url: 'https://michaelcalewaert.netlify.app/',
```

Comment:
This needs to be updated - Can you update our Arcus Background Jobs site in Netlify please to work with this?

Reply:
Can you help me with this? Only site I see is the `background-jobs.staging`

Reply:
That's the one indeed, we'll need to move our DNS record there once this is merged.